packages/manager/apps/pci-vouchers/vitest.config.js

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
```

Comment:
i think it's better for each uapp to have it's own jest config / version (or vitest instead of jest) ; that way if you need to update jest (version or config) you can do it incrementally per application

having a global jest config will be hard to maintain because you'll have to update all the impacted apps ; right now it's not an issue since we have few uapp but it'll become difficult to maintain over time